toast-tk-runtime/src/main/java/com/synaptix/toast/runtime/TestClass.java

```diff
@@ -0,0 +1,12 @@
+package com.synaptix.toast.runtime;
+
+public class TestClass {
+
+	public TestClass() {
+		String test = null;
+		int length = test.length(); // sonar preview should see this error and fail the build
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "length" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS1481)